test/spec/functional/listView.js

```diff
@@ -398,6 +404,84 @@ describe('A List View', function() {
     expect(endTime < threshold).toBe(true);
   });
 
+  it('can reset a few of many models correctly and in a reasonable time', function() {
+    var startTime, endTime, i,
+      models = [],
+      numberOfViews = 1000,
+      threshold = 500;
+    for (i = 0; i < numberOfViews; i++) {
+      models.push(new Model())
+    }
+    myListView.dispose();
```

Comment:
why define list view globally if you always dispose it.  Seems like extra work, instead of a list view inside of each it method.